datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1322,6 +1406,23 @@ func completeSyntheticsTestRequest(request datadogV1.SyntheticsTestRequest, requ
 		request.SetCertificate(requestClientCertificate)
 	}
 
+	if len(requestProxy) > 0 {
+		if proxy, ok := requestProxy[0].(map[string]interface{}); ok {
+			requestProxy := datadogV1.SyntheticsTestRequestProxy{}
```

Comment:
Could we rename the `requestProxy` variable here so it doesn't clash with the one from outside scope